_src/static/sass/components/_buttons.scss

```diff
@@ -259,33 +265,13 @@
 // #LINK
 // ----------------------------
 .btn-link {
```

Comment:
Hmm, I was hoping that could be done through ``display: inline-block;`` and ``vertical-align: middle;`` when we're doing the more complex patterns, like form actions or other places where we'd use buttons side-by-side. Cool to wait till then?